taprpc/tapchannelrpc/tapchannel.proto

```diff
@@ -181,3 +189,30 @@ message AddInvoiceResponse {
     // The result of the invoice creation.
     lnrpc.AddInvoiceResponse invoice_result = 2;
 }
+
+message AssetPayReqString {
+    // The asset ID that will be used to resolve the invoice's satoshi amount.
+    bytes asset_id = 1;
+
+    // The normal LN invoice that whose amount will be mapped to units of the
+    // asset ID.
+    lnrpc.PayReqString pay_req_string = 2;
```

Comment:
why not `string` directly? IIUC `PayReqString` is just there because one can't use `string` as the request message type for an RPC method directly.